fetch/worker.go

```diff
@@ -79,7 +80,10 @@ func getBlocksFromBatch(chunkRange chunkRange, client Client) ([]*types.Block, e
 	}
 
 	// Get the block results
-	blocksRaw, err := batch.Execute()
+	ctx, cancelFn := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
shall we make this configurable? If you prefer to do that on another PR, maybe we should then keep this context as a `context.Background()` to keep the previous behavior.

Reply:
You're right, we should probably make this configurable 🤦‍♂️ 

Swapped:
[46fb3cb](https://github.com/gnolang/tx-indexer/pull/41/commits/46fb3cb89458580fecd40c58f4b6db2ae4459b32)